doc-requirements.txt

```diff
@@ -118,14 +114,18 @@ numpy==1.22.2
     #   pyarrow
 packaging==21.3
     # via sphinx
-pandas==1.4.0
+pandas==1.4.1
     # via flytekit
 poyo==0.5.0
     # via cookiecutter
 protobuf==3.19.4
     # via
     #   flyteidl
     #   flytekit
+    #   googleapis-common-protos
+    #   protoc-gen-swagger
+protoc-gen-swagger==0.1.0
```

Comment:
Can you make sure your changes are in sync with the master branch because this library is already available in master?